security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,6 +12,24 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1-14-1"]
+== {cert-manager-operator} 1.14.1
+
+Issued: 2024-10-17
+
+The following advisory is available for the {cert-manager-operator} 1.14.1:
+
+* link:https://errata.devel.redhat.com/advisory/140037[RHEA-2024:140037]
```

Comment:
IIRC, this has to be an access.redhat.com link for external customers to be able access the info.

Reply:
Yep, once the advisory is done, I will update the link.